app/components/dex-create.jsx

```diff
@@ -7,14 +7,29 @@ import slug          from 'slug';
 import { AlertComponent } from './alert';
 import { ReactGA }        from '../utils/analytics';
 import { createDex }      from '../actions/dex';
+import { listGames }      from '../actions/games';
 
 const NATIONAL_ONLY_GAMES = ['x', 'y', 'omega_ruby', 'alpha_sapphire'];
 
 export class DexCreate extends Component {
 
   constructor (props) {
     super(props);
-    this.state = { error: null, game: 'sun', regional: false };
+    this.state = { error: null, game: 'ultra_moon', games: [], regional: false };
```

Comment:
ive changed the order returned from api to be

```
us
um
sun
moon
or
as
x
y
```

so this should be `game: 'ultra_sun'`